UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Helpers/CommonHelper.cs

```diff
@@ -63,5 +70,29 @@ public static long GetFileSize(SalesCatalogueProductResponse salesCatalogueRespo
             }
             return fileSize;
         }
+
+        public static IAsyncPolicy<HttpResponseMessage> GetRetryPolicy(ILogger logger, string requestType, EventIds eventId, int retryCount, double sleepDuration)
```

Comment:
Kindly add unit test case for this method.

Reply:
Yes added Unit Test for common helper method